internal/isoeditor/rhcos.go

```diff
@@ -247,6 +247,12 @@ func (e *rhcosEditor) addCustomRAMDisk(clusterISOPath, staticNetworkConfig strin
 		return err
 	}
 
+	// Ensures RAM placeholder is large enough to accommodate the custom disk
+	if uint64(buffer.Len()) > RamDiskPaddingLength {
```

Comment:
This should use the info from the iso rather than the const.

Reply:
right, done.